DependencyInjection/Configuration.php

```diff
@@ -70,8 +70,8 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                     ->prototype('array')
                         ->children()
                             ->scalarNode('subject')->isRequired()->cannotBeEmpty()->end()
-                            ->scalarNode('attribute')->isRequired()->defaultValue(Attribute::class)->cannotBeEmpty()->end()
-                            ->scalarNode('option')->isRequired()->defaultValue(Option::class)->cannotBeEmpty()->end()
+                            ->scalarNode('attribute')->isRequired()->defaultValue(ProductAttribute::class)->cannotBeEmpty()->end()
+                            ->scalarNode('option')->isRequired()->defaultValue(ProductOption::class)->cannotBeEmpty()->end()
```

Comment:
We don't need the variation bundle anymore, so what you say is basically merging Product into Archetype. And to me, Archetype is strongly linked with Product anyway, so it makes sense to have it as a dependency..